README.md

```diff
@@ -56,4 +56,148 @@ We may come back to this effort in the future, but we don’t want to stop the c
+
+## Getting Started
+
+> [!NOTE]
+> InfluxDB Edge is pre-release and in active development. These usage instructions are only designed for a very quick start and subject to change.
+
+The following assumes that `influxdb3` exists in your current directory. Please adjust as necessary for your environment. Help can be seen with `influxdb3 --help`.
+
+### Starting the server
+
+`influxdb3` has many different configuration options, which may be controlled via command line arguments or environment variable. See `influxdb3 serve --help` for details.
+
+Eg, to start the server with a local filesystem object store (default listens on port 8181 with no authentication):
+```
+$ export INFLUXDB_IOX_OBJECT_STORE=file
+$ export INFLUXDB_IOX_DB_DIR=/path/to/influxdb3
+$ influxdb3 serve
+2024-05-10T15:54:24.446764Z  INFO influxdb3::commands::serve: InfluxDB3 Edge server starting git_hash=v2.5.0-14032-g1b7cd1976d65bc7121df7212cb234fca5c5fa899 version=0.1.0 uuid=3c0e0e61-ae4e-46cb-bba3-9ccd7b5feecd num_cpus=20 build_malloc_conf=
+2024-05-10T15:54:24.446947Z  INFO clap_blocks::object_store: Object Store db_dir="/path/to/influxdb3" object_store_type="Directory"
+2024-05-10T15:54:24.447035Z  INFO influxdb3::commands::serve: Creating shared query executor num_threads=20
+...
+```
+
+### Interacting with the server
+The `influxdb3` binary is also used as a client.
+
+#### Writing
+```
+# write help
+$ influxdb3 write --help
+...
+
+# create some points in a file for the client to write from using line protocol
+$ cat > ./file.lp <<EOM
+mymeas,mytag1=sometag value=0.54 $(date +%s%N)
+mymeas,mytag1=sometag value=0.55 $(date +%s%N)
+EOM
+
+# perform the write
+$ influxdb3 write --dbname mydb -f ./file.lp
+success
+```
+
+See the [InfluxDB documentation](https://docs.influxdata.com/influxdb/cloud-dedicated/reference/syntax/line-protocol/) for details on line protocol.
+
+`curl` can also be used with the `/api/v3/write_lp` API (subject to change):
+```
+$ export URL="http://localhost:8181"
+$ curl -s -X POST "$URL/api/v3/write_lp?db=mydb" --data-binary @file.lp
+```
+
+#### Querying
+```
+# query help
+$ influxdb3 query --help
+...
+
+# perform a query with SQL
+$ influxdb3 query --dbname mydb "SELECT * from mymeas"
++------------------------------------------------------------------+---------+-------------------------------+-------+
+| _series_id                                                       | mytag1  | time                          | value |
++------------------------------------------------------------------+---------+-------------------------------+-------+
+| 05a045398acb0754a6551b74ca09d180d7e452d72b4f196da665490a6dd9e788 | sometag | 2024-05-09T21:08:52.227359715 | 0.54  |
+| 05a045398acb0754a6551b74ca09d180d7e452d72b4f196da665490a6dd9e788 | sometag | 2024-05-09T21:08:52.229494943 | 0.55  |
++------------------------------------------------------------------+---------+-------------------------------+-------+
```

Comment:
With https://github.com/influxdata/influxdb/pull/24969 we ripped out the `_series_id`, so with the most recent `main` that column will be removed, and will tidy this up a bit.

Reply:
Ok, removed. Thanks!